CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+**Features**:
+- Provide the option to attach a view hierarchy file. ([#1191](https://github.com/getsentry/sentry-native/pull/1191))
```

Comment:
Should the Changelog also point to the Relay update (https://github.com/getsentry/relay/pull/4624)?

Reply:
It is okay if it's only visible from the PR. I only add our forked/vendored dependencies to the changelog. But you can add it if you'd like to.